### DISCUSSION TITLE
invoices not working in demo

### DISCUSSION THREAD

**Discussion by derStephan** [kimai/kimai#4134] [General]
### Describe the issue

I wanted to test kimai on official demo but received an error  500. I tried that a couple of days ago and hoped the problem disappeared by now. But it did not. 

![grafik](https://github.com/kimai/kimai/assets/7764931/1d9e19a4-1722-4073-a831-9030597d9204)

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

demo

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.1

**Comment by derStephan** [comment]
May I ask why this is a discussion? Is there a hard line to be crossed before it is a valid bug?

**Reply by kevinpapst** [reply]
Invoices work. So it is not a bug

**Reply by derStephan** [reply]
Ok, but wouldn't that be simply a status in the bug? Like "can't reproduce"? Anyways, thanks for the help. Selecting only one month did the trick.

**Comment by kevinpapst** [comment]
Works for me without any issue using this month and clicking preview for any displayed customer.

I see an "out of memory" error in the logs. 
Maybe you try to render an invoice for one month only. 
This is still a demo server and does not get that much RAM ...

Or you tell me, what exactly you are trying to do.

**Comment by derStephan** [comment]
I did not know that there was a limitation. Sorry. I just clicked around and used the very first option in the time range 

![grafik](https://github.com/kimai/kimai/assets/7764931/245f0265-3319-45b2-bc38-528b12315fe8)

I did not know that this was the problem. Sorry for the noise.

**Reply by kevinpapst** [reply]
Oh, yeah, Kimai was loading more than 11k entries, which was a bit much probably 😁
